starter/script.js

```diff
@@ -1 +1,34 @@
-// put code here!
+confirm("This game contains strong language and some adult situations. Player discretion is advised...")
```

Comment:
Pseudocode can help us plan our programs; that should go here according to the assignment